netstandard/src/netstandard.csproj

```diff
@@ -40,26 +40,36 @@
   </ItemGroup>
   <ItemGroup Condition="'$(TargetGroup)'=='xamarin.ios'">
```

Comment:
You can eliminate this duplication by changing the condition to `$(TargetGroup).StartsWith('xamarin')`